content/en/integrations/guide/storage-monitoring-setup.md

```diff
@@ -0,0 +1,163 @@
+
+   Optional parameters:
+   - **SourceBucketPrefix**: (Optional) Limit monitoring to a specific path in the source bucket
+
+{{< img src="integrations/guide/storage_monitoring/specify_stack_details.png" alt="Specify stack details" responsive="true" style="width:90%;" >}}
+
+7. Click **Next**.
+8. Wait for AWS to locate your source bucket, and click **Import resources** in the bottom right corner.
+
+**Note:** This CloudFormation template can be rolled back, but rolling back doesn't delete the created resources. This is to ensure the existing bucket doesn't get deleted. You can manually delete the inventory configurations by going on the **Management** tab in the bucket view.
+
+**Note:** Review [Amazon S3 pricing][106] for costs related to inventory generation.
+#### Step 2: Configure required permissions
+
+This template creates two IAM policies:
+  - A policy to allow Datadog to read inventory files from the destination bucket
+  - A policy to allow your source bucket to write inventory files to the destination bucket
+
+1. Download the [cloud-inventory-policies-cfn.yaml][103] template.
+2. In [AWS CloudFormation][104], click **Create stack** in the top right corner and select **With new resources (standard)**.
+3. In the **Specify template** step, select **Upload a template file**.
+4. Click **Choose file** and select the `cloud-inventory-policies-cfn.yaml` file, then click **Next**.
+5. Fill in the required parameters:
+   - **DatadogIntegrationRole**: Your Datadog AWS integration role name
+   - **DestinationBucketName**: The name of the bucket that to receive your inventory files. **Note**: You must only use one destination bucket for all inventory files generated in an AWS account.
+   - **SourceBucketName**: The name of the bucket you want to start generating inventory files for
+   - **DestinationBucketPrefix**: If you want to reuse an existing bucket as the destination, this parameter allows the inventory files to be shipped to a specific prefix in that bucket. Ensure that any prefixes do not include trailing slashes (`/`)
+
+   Optional parameters:
+   - **SourceBucketPrefix**: This parameter limits the inventory generation to a specific prefix in the source bucket
+
+{{< img src="integrations/guide/storage_monitoring/bucket_policy_stack_details.png" alt="Stack parameters for bucket policy" responsive="true" style="width:90%;" >}}
+
+6. On the **Review and create** step, verify the parameters have been entered correctly, and click **Submit**.
+
+#### Post-setup steps
+
+After completing the CloudFormation setup, [reach out][105] with the following information:
+    1. Name of the destination bucket holding the inventory files.
+    2. Prefix where the files are stored in the destination bucket.
+    3. Name of the source bucket you want to monitor (the bucket producing inventory files).
+    4. AWS region of the destination bucket holding the inventory files.
+    5. AWS account ID containing the buckets.
+    6. Datadog org ID.
```

Comment:
In the preview, this list isn't getting formatted correctly. Let's try removing the indentation and see if that fixes it.
```suggestion
1. Name of the destination bucket holding the inventory files.
2. Prefix where the files are stored in the destination bucket.
3. Name of the source bucket you want to monitor (the bucket producing inventory files).
4. AWS region of the destination bucket holding the inventory files.
5. AWS account ID containing the buckets.
6. Datadog org ID.
```